weave

```diff
@@ -926,10 +926,8 @@ proxy_args() {
 proxy_addr() {
     if addr=$(docker logs $PROXY_CONTAINER_NAME 2>/dev/null | head -n3 | grep -oE "proxy listening on .*"); then
       addr=${addr##* }
-      host=${addr%:*}
-      [ "$host" = "0.0.0.0" ] && host=$PROXY_HOST
-      port=${addr#*:}
-      echo "${1}tcp://${host:-$PROXY_HOST}:${port:-$PROXY_PORT}"
+      addr=$(echo ${addr} | sed "s/0.0.0.0/$PROXY_HOST/g")
+      echo "${1}${addr}"
```

Comment:
or instead of three lines just have

`echo "${1}${addr##* }" | sed "s/0.0.0.0/$PROXY_HOST/"`

(no need for the `g`)